### PR TITLE
#186687029 : Added example for .NET8

### PR DESCRIPTION
**Pull request by praves77**

### CODE REVIEW

**Comment by praves77** [comment]
Steps involved :
- I cloned MoesifNet6Example as MoesifNet8Example from our [net core example repo](https://github.com/Moesif/moesif-netcore-example)
- Renamed MoesifNet8Example/`MoesifNet6Example.csproj` to MoesifNet8Example/`MoesifNet8Example.csproj`
- Updated project's framework to .net8 as seen in the screenshot below and it worked.
Screenshot 2023-12-14 at 7 10 19 PM

File: MoesifNet8Example/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Moesif .Net 6 Example
```
Comment (keyur9) [inline MoesifNet8Example/README.md]: there are several places in this file where it needs to be changed from Net6 to Net8

File: MoesifNet8Example/Settings/MoesifOptions.cs [hunk 2]
```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Microsoft.AspNetCore.Http;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.Extensions.Configuration;
+
+namespace MoesifNet6Example.Settings
```
Comment (keyur9) [inline MoesifNet8Example/Settings/MoesifOptions.cs]: we'd need to change the namespace from Net6 to Net8

File: MoesifNet8Example/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Moesif.Middleware;
+using MoesifNet6Example.Settings;
+using Microsoft.AspNetCore.Routing;
+
+namespace MoesifNet6Example
```
Comment (keyur9) [inline MoesifNet8Example/Startup.cs]: we'd need to change the namespace from Net6 to Net8

File: MoesifNet8Example/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,65 @@
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Moesif.Middleware;
+using MoesifNet6Example.Settings;
```
Comment (keyur9) [inline MoesifNet8Example/Startup.cs]: we'd need to change the import from Net6 to Net8

**Review by keyur9 — CHANGES_REQUESTED** [review]

**Comment by praves77** [comment]
@keyur9 updated PR

**Pull request merged**